IaC/BareMetal/README.md

```diff
@@ -38,116 +44,37 @@ az monitor log-analytics workspace create -g $Ngsa_Log_RG -n $Ngsa_Log_Name -l $
 
 ```
 
-## Create VM
-
-Create your VM per instructions in [Bare Metal Setup](setup-bare-metal-vm.md)
-
-- It should work on any Ubuntu 18.04 VM
-- We've tested on
-  - Azure VMs
-  - Multipass VMs
-  - Digital Ocean VMs
-
-## setup k8s
-
-### Export your public IP address
-
-```bash
-
-# check if it's set correctly
-echo $PIP
-
-# set if necessary
-export PIP=YourPublicIPAddress
-
-```
-
-### Reset cluster if necessary
-
-```bash
-
-# reset your cluster
-sudo kubeadm reset -f
+## Create Azure VM
 
-```
+Create your Azure VM per instructions at [Azure Kubernetes Development Cluster](https://github.com/retaildevcrews/akdc)
 
-### Initialize cluster
+## SSH into the Azure VM
 
 ```bash
 
-# make sure you're in the ngsa/IaC/BareMetal directory
-cd IaC/BareMetal
-
-# make sure public IP is set correctly
-echo $PIP
-
-# install k8s controller
-sudo kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address $PIP
-
-### WARNING ###
-# This will delete your existing kubectl configuration
-# Make sure to back up or merge manually
-###############
-
-# setup your config file
-sudo rm -rf $HOME/.kube
-mkdir -p $HOME/.kube
-sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
-sudo chown -R $(id -u):$(id -g) $HOME/.kube
+# AKDC_IP is set during the previous step
+# the -L allows you to forward your port from the dev cluster via SSH tunneling
+ssh -L 4120:127.0.0.1:4120 akdc@${AKDC_IP}
 
-# add flannel network overlay
-kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yaml --namespace=kube-system
+# clone this repository
+git clone https://github.com/retaildevcrews/ngsa
 
-# optional - add the taint to schedule normal pods on the control plane
-# this let you run a "one node" cluster for `development`
-k taint nodes --all node-role.kubernetes.io/master-
+# change to the correct directory
+cd ngsa/IaC/BareMetal/app
 
-# patch kube-proxy for metal LB
```

Comment:
do we need metal lb in the dev cluster? I very seldom use it

Reply:
I like the idea of making it an optional step. Maybe even adding example `loadbalancer` service.

Reply:
oops, nevermind. I see the existing loadbalancer.